SWLOR.Game.Server/Enumeration/PerkType.cs

```diff
@@ -90,7 +90,7 @@ public enum PerkType
         ScavengingExpert = 85,
         ForceInsight = 86,
         WeaponFocusLightsaber = 87,
-        // 88 is free
+        SpeedyCrafting = 88,
```

Comment:
Let's get rid of the individual speedy crafting perks in this file too. Make sure to add the deletes to the perk info in the DB migration script(s) as well.